transifex/requirements.txt

```diff
@@ -1,5 +1,8 @@
+Babel==1.3
 Django==1.11.2
+django-babel-underscore==0.5.2
 edx-i18n-tools==0.3.9
+mako==1.0.7
```

Comment:
Please update these requirements in a separate PR. This is not related to the original purpose of this PR, which was to simply add the --repo-owner flag. It should be reviewed and tested separately.